src/applications/vaos/tests/new-appointment/components/DateTimeSelectPage/index.unit.spec.jsx

```diff
@@ -1,280 +1,264 @@
-        appointmentSlotsStatus={FETCH_STATUS.succeeded}
-      />,
-    );
-
-    form
-      .find('FormButtons')
-      .find('button[type="submit"]')
-      .simulate('click');
-    expect(routeToNextAppointmentPage.called).to.be.false;
-    form.unmount();
-  });
-
-  it('should not submit form with validation error', () => {
-    const getAppointmentSlots = sinon.spy();
-    const onCalendarChange = sinon.spy();
-    const routeToNextAppointmentPage = sinon.spy();
+  it('should allow user to select date and time for a VA appointment', async () => {
+    const store = createTestStore({
+      newAppointment: {
+        availableSlots,
+        data: {
+          calendarData: {
+            currentlySelectedDate: null,
+            selectedDates: [],
+          },
+          preferredDate: getMondayTruFriday(moment().add(5, 'd')),
+          vaParent: 'var983',
+          clinicId: 'var_408',
+        },
+        pages: [],
+        previousPages: [],
+        eligibility: [],
+        parentFacilities: parentFacilitiesParsed,
+      },
+    });
 
-    const form = mount(
-      <DateTimeSelectPage
-        getAppointmentSlots={getAppointmentSlots}
-        onCalendarChange={onCalendarChange}
-        data={{
-          calendarData: { currentlySelectedDate: '2020-12-20' },
-        }}
-        availableDates={availableDates}
-        facilityId="123"
-        availableSlots={availableSlots}
-        routeToNextAppointmentPage={routeToNextAppointmentPage}
-        appointmentSlotsStatus={FETCH_STATUS.succeeded}
-      />,
+    const screen = renderWithStoreAndRouter(
+      <Route component={DateTimeSelectPage} />,
+      {
+        store,
+      },
     );
 
-    form
-      .find('FormButtons')
-      .find('button[type="submit"]')
-      .simulate('click');
-    expect(routeToNextAppointmentPage.called).to.be.false;
-    form.unmount();
+    // it should display page heading
+    expect(
+      await screen.findByRole('heading', {
+        level: 1,
+        name: /Tell us the date and time you’d like your appointment/i,
+      }),
+    ).to.be.ok;
+
+    // it should display 2 calendar months for VA appointments
+    expect(
+      screen.getByRole('heading', {
+        level: 2,
+        name: moment().format('MMMM YYYY'),
+      }),
+    ).to.be.ok;
+    expect(
+      screen.getByRole('heading', {
+        level: 2,
+        name: moment()
+          .add(1, 'M')
+          .format('MMMM YYYY'),
+      }),
+    ).to.be.ok;
+
+    // Find all available appointments for the current month
+    const currentMonth = moment().format('MMMM');
+    const buttons = screen
+      .getAllByRole('button', { name: new RegExp(`${currentMonth}`) })
+      .filter(button => button.disabled === false);
+
+    // it should allow the user to select an appointment time
+    userEvent.click(buttons[0]);
+    const time = moment()
+      .minute(0)
+      .format('h:mm');
+    const radio = screen.getByRole('radio', {
```

Comment:
I think we want to have an await here when picking the radio button to choose. 

I had to do that here for a similar test: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/tests/new-appointment/components/DateTimeSelectPage/select-date-page.unit.spec.jsx#L146